### ISSUE TITLE
Dashboard does not show running containers

### ISSUE THREAD

**Issue by hoegge** [docker/for-win#6037]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: EFD34D26-FC8E-49AF-92C6-25B3A8A19B8A/20200316100129

### Expected behavior
Open dashboard and see running containers

### Actual behavior
None are shown - if you use `docker ps` you see them.

### Information

  - Windows Version: 10 / 1909 / build 18363.719
  - Docker Desktop Version: 2.2.0.4.(43472)
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: no

### Steps to reproduce the behavior

Is not always the case about 50% of the times. If you restart docker the dashboard often shows the containers again.

**Comment by mat007** [comment]
Hi, thanks for the report.
We have found and fixed an issue which sounds suspiciously similar to what you’re describing.
Here is a link to a private build to install an Edge version that fixes it, if you would like to try it.

Or if you wanted to keep your current version you could have a look at your `~/.kube/config` file and check if you have entries without `cluster` or `user` properly filled for some reason, e.g.
```
- context:
    cluster:
    user:
  name: some-context-name
```
If you were to remove them, that should hopefully fix the issue.

**Comment by stephen-turner** [comment]
Hopefully the thumbs up on the last comment means that this is fixed! We released the fix in 2.3 today, so closing this ticket.
